src/brain_atlas/util/hail_batch/14_hail_docker/getProgress.sh

```diff
@@ -0,0 +1,14 @@
+cd /dev/shm
+
+while true; do
+    if compgen -G "/dev/shm/rtmp*" > /dev/null; then
+        echo "Some files exist."
+        break
+    fi
+    sleep 30
+done
+
+TOTAL=$(ls|grep rtmp|head -n 1|sed -E "s/(^[^_]*_)|(__.*)//g")
+echo $TOTAL
+# Assume divide by 10
+while true; do ls rtmp*|wc -l; sleep 10; done|tqdm --ncols=100  --update_to --total=$TOTAL --smoothing=0.3 --initial=`ls rtmp* |wc -l`
```

Comment:
What does this script do? It doesn't look like it's getting used, but maybe you use it interactively?

Reply:
Oh sorry, I meant to remove it, it just makes a nice cli progress bar (I used it for the mapping hail). Could be marginally nice if you end up running multiple diffexps per hail batch, just to track the progress, but unecessary

Reply:
I guess I just removed it from the Dockerfile, it was called with a background ampersand same as getcpu.sh